common/djangoapps/course_modes/tests/test_models.py

```diff
@@ -471,3 +471,22 @@ def test_verified_min_price(self, mode_slug, is_error_expected):
             self.assertTrue(is_error_expected, "Did not expect a ValidationError to be thrown.")
         else:
             self.assertFalse(is_error_expected, "Expected a ValidationError to be thrown.")
+
+
+class EnrollmentModeTest(TestCase):
+    """
+    Tests for EnrollmentMode Model.
+    """
+    def tearDown(self):
+        super(EnrollmentModeTest, self).tearDown()
+        EnrollmentMode.objects.all().delete()
```

Comment:
@cahrens: @nasthagiri's right. As long as you're inheriting from Django's `TestCase`, individual test methods will be run in a transaction and rolled back at the end, so:

```
start transaction
setUp()
test_method_1()
tearDown()
rollback

start transaction
setUp()
test_method_2()
tearDown()
rollback

etc.
```